tests/Backend/Snowflake/TableDefinitionOperationsTest.php

```diff
@@ -77,6 +79,712 @@ public function testResponseDefinition(): void
+                    'expectFail' => [],
+                ],
+                'empty' => [
+                    'value' => '',
+                    'expectFail' => [],
+                ],
+                'fail quoted' => [
+                    'value' => '\'2022-02-22\'',
+                    'expectFail' => [
+                        'message' => 'Table creation ended with a syntax exception, probably due to an invalid "default" column specification. Original exception is:',
+                    ],
+                ],
+                'fail type' => [
+                    'value' => 'string',
+                    'expectFail' => [
+                        'message' => 'Table creation ended with a syntax exception, probably due to an invalid "default" column specification. Original exception is:',
+                    ],
+                ],
+            ],
+            'DATETIME' => [
+                'working' => [
+                    'value' => 'current_timestamp()',
+                    'expectFail' => [],
+                ],
+                'empty' => [
+                    'value' => '',
+                    'expectFail' => [
+                    ],
+                ],
+                'fail quoted' => [
+                    'value' => '\'2021-01-01 00:00:00\'',
+                    'expectFail' => [
+                        'message' => 'Table creation ended with a syntax exception, probably due to an invalid "default" column specification. Original exception is:',
+                    ],
+                ],
+                'fail not quoted' => [
+                    'value' => '2021-01-01 00:00:00',
+                    'expectFail' => [
+                        'message' => 'Table creation ended with a syntax exception, probably due to an invalid "default" column specification. Original exception is:',
+                    ],
+                ],
+                'fail type' => [
+                    'value' => 'string',
+                    'expectFail' => [
+                        'message' => 'Table creation ended with a syntax exception, probably due to an invalid "default" column specification. Original exception is:',
+                    ],
+                ],
+            ],
+            'TIME' => [
+                'working' => [
+                    'value' => 'current_time()',
+                    'expectFail' => [],
+                ],
+                'empty' => [
+                    'value' => '',
+                    'expectFail' => [
+                    ],
+                ],
+                'fail not quoted' => [
+                    'value' => '00:00:00',
+                    'expectFail' => [
+                        'message' => 'Table creation ended with a syntax exception, probably due to an invalid "default" column specification. Original exception is:',
+                    ],
+                ],
+                'fail quoted' => [
+                    'value' => '\'00:00:00\'',
```

Comment:
Tohle by mělo projít, ne? 

Aha, tak ne. Musí se to udělat takhle:

    CREATE OR REPLACE TABLE mytable (id boolean DEFAULT 'test', cas TIME DEFAULT '00:00:00'::time, num number);